proto/Makefile

```diff
@@ -26,9 +26,17 @@ clean:
 	rm -fR gen
 	rm -fR dist
 
+.PHONY: deps
+deps: go-deps py-deps
+
+.PHONY: go-deps
+go-deps:
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+
 .PHONY: py-deps
 py-deps:
```

Comment:
this should be run in venv, since on most distributions pip won't allow install in user dir.
I suggest to use poetry or venv we are using for python testing

Reply:
plz open ticket to fix py-deps

Reply:
there is a ticket to use buf to generate python binding for remote runtime, 
I think once it works I can remove all of this dependencies

I'll add a reference in the ticket to this discussion